server/data/vocabularies.json

```diff
@@ -0,0 +1,2246 @@
+                "name": "Thailand",
+                "is_active": true
+            },
+            {
+                "qcode": "TLS",
+                "name": "Timor-Leste",
+                "is_active": true
+            },
+            {
+                "qcode": "TGO",
+                "name": "Togo",
+                "is_active": true
+            },
+            {
+                "qcode": "TKL",
+                "name": "Tokelau",
+                "is_active": true
+            },
+            {
+                "qcode": "TON",
+                "name": "Tonga",
+                "is_active": true
+            },
+            {
+                "qcode": "TTO",
+                "name": "Trinidad and Tobago",
+                "is_active": true
+            },
+            {
+                "qcode": "TUN",
+                "name": "Tunisia",
+                "is_active": true
+            },
+            {
+                "qcode": "TUR",
+                "name": "Turkey",
+                "is_active": true
+            },
+            {
+                "qcode": "TKM",
+                "name": "Turkmenistan",
+                "is_active": true
+            },
+            {
+                "qcode": "TCA",
+                "name": "Turks and Caicos Islands",
+                "is_active": true
+            },
+            {
+                "qcode": "TUV",
+                "name": "Tuvalu",
+                "is_active": true
+            },
+            {
+                "qcode": "UGA",
+                "name": "Uganda",
+                "is_active": true
+            },
+            {
+                "qcode": "UKR",
+                "name": "Ukraine",
+                "is_active": true
+            },
+            {
+                "qcode": "ARE",
+                "name": "United Arab Emirates",
+                "is_active": true
+            },
+            {
+                "qcode": "GBR",
+                "name": "United Kingdom",
+                "is_active": true
+            },
+            {
+                "qcode": "USA",
+                "name": "United States of America",
+                "is_active": true
+            },
+            {
+                "qcode": "UMI",
+                "name": "US Minor Outlying Islands",
+                "is_active": true
+            },
+            {
+                "qcode": "URY",
+                "name": "Uruguay",
+                "is_active": true
+            },
+            {
+                "qcode": "UZB",
+                "name": "Uzbekistan",
+                "is_active": true
+            },
+            {
+                "qcode": "VUT",
+                "name": "Vanuatu",
+                "is_active": true
+            },
+            {
+                "qcode": "VEN",
+                "name": "Venezuela (Bolivarian Republic)",
+                "is_active": true
+            },
+            {
+                "qcode": "VNM",
+                "name": "Viet Nam",
```

Comment:
I think it should be just a Vietnam, not a Viet Nam.

Reply:
hum, it's a copy/paste from AP, so it needs to be fixed there too, thx for the catch.